### ISSUE TITLE
Add "single-instance" to "load-balanced" upgrade button

### ISSUE THREAD

**Issue by axelfontaine** [boxfuse/boxfuse-issues#139]
In Boxfuse Console, one button per applicable app.

**Comment by forbode** [comment]
This would be nice. I'll likely be doing this in the near future. One requirement would be to use the existing Elastic IP for the load balancer endpoint as well. In the customers I deal with often we have IP controlled access to certain machines. If the IP changes then we have to get all sorts of approval to allow the new IP to access a machine. I do a lot of automation so M2M important to me.

**Comment by axelfontaine** [comment]
@forbode Reusing an EIP for an ELB is unfortunately not possible. An ELB usually has 2 or more IP addresses which may change at AWS's discretion. The only solution to this would be to have the machine behind the EIP act as a proxy for the ELB.

**Comment by forbode** [comment]
Good point yes. Thanks for the clarity. Either way it would be good to have a conversion button.
